### DISCUSSION TITLE
How to complete remove (purge) unlighthouse ?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [harlan-zw/unlighthouse#238] [Q&A] [answered]
OS: Arch linux

i already remove directory
/home/user/.npm/_npx/
command unlighthouse still exist

![image](https://github.com/user-attachments/assets/c4bc91c0-a08c-4668-8b94-9c3db0658642)
![image](https://github.com/user-attachments/assets/74abc93e-5a1b-4341-b95b-0dc520d89533)

**Comment by mathiasrw** [comment, marked as answer]
Whatever the issue, it has nothing to do with unlighthouse.

I'm wondering if you installed it with something other than npm - Yarn maybe?

In any case, run `which unlighthouse` to see where the bin is located and then `rm` it.
